packages/ui/src/Elevation/Elevation.tsx

```diff
@@ -19,15 +19,12 @@ export type ElevationProps = {
 };
 
 /**
- * Elevation component visually raises any content by applying shadow.
+ * @deprecated This component is deprecated and will be removed in future releases.
+ * Please use the `Card` component from the `@webiny/admin-ui` package instead.
  */
-const Elevation = React.forwardRef((props: ElevationProps, ref: React.Ref<any>) => {
-    return (
-        <RmwcElevation ref={ref} {...props}>
-            {props.children}
-        </RmwcElevation>
-    );
-});
+const Elevation = (props: ElevationProps) => {
+    return <Card {...props} content={<CardContent>{props.children}</CardContent>} />;
```

Comment:
I would create an `Elevation` component in the new `admin-ui` package and reference it here and inside the `Card` component.

Reply:
This one I didn't quite understand 🤔 

What's interesting is that now there's an `elevation` prop on Card, so maybe we don't need a separate `Elevation` component?